### DISCUSSION TITLE
Zscaler corporate proxy and CDK deploy issue

### DISCUSSION THREAD

**Discussion by jasonmlehman** [aws/aws-cdk#25785] [Q&A]
Downloaded the latest version of the AWS CLI and CDK.  Configure AWS SSO accordingly.

When I attempted to deploy the stack it fails and I get an SSL issue.  Exact error is:

demo-pipeline-dev:  fail: unable to get local issuer certificate

Does anyone know of a way to get this to work with Zscaler? 

The AWS CLI seems to work fine, no SSL error's, it's just CDK.

Thanks!

**Comment by indrora** [comment]
Is there further context to the error? Please include, e.g., log information and some more details on your setup. Does this happen when not behind ZScaler?
